docs/dev/apis/configurable-company/developing-configs.md

```diff
@@ -0,0 +1,112 @@
+---
+prev: true
+next: true
+description: How to create configurations for your plugin and how to use them.
```

Comment:
add 'Configurable Company' to this
...create Configurable Company configurations...